pkg/hoist/hoist_launchable.go

```diff
@@ -239,30 +233,12 @@ func (hl *Launchable) Install() error {
 		return nil
 	}
 
-	outDir, err := ioutil.TempDir("", hl.Version())
-	defer os.RemoveAll(outDir)
-	if err != nil {
-		return util.Errorf("Could not create temporary directory to install %s: %s", hl.Version(), err)
-	}
-
-	outPath := filepath.Join(outDir, hl.Version())
-
-	err = hl.FetchToFile(hl.Location, outPath)
+	data, err := hl.Fetcher.Open(hl.Location)
```

Comment:
This is a neat simplification. Keep in mind that we want *some* strategy to allow caching across multiple pods if we share launchables. A shared jdk artifact is one example. We can't totally guarantee what will happen to an install directory after expansion unless we do some magic, so the tar was going to be the next logical step.